cre.cpp

```diff
@@ -1914,7 +1914,10 @@ static int getTextFromPositions(lua_State *L) {
 		// It's usually OK with multiple chars selection, even if one needs to go over the further
 		// 2nd half of the glyph to have it included in the selection.
 		// But it is less OK with an initial long-press on a glyph, where we get a 50% chance of
-		// having the next char selected. Try to handle this case better:
+		// having the next char selected. Try to handle this case better.
+		// It turns out this also happens when some CJK/symbol char is near a multi-alpha word,
```

Comment:
multi-apha word?

Reply:
How you would name a word with more than one alphanumerical chars ? vs. CJK and symbols &copy; which are technically words in this context of text selection.

Reply:
Not complaining, just not sure what it meant.

Maybe multicharacter?